ex03.ex

```diff
@@ -80,8 +89,13 @@ defmodule Ex03 do
       true
 
   """
+  def list_contains([], _val), do: false
```

Comment:
# -1

This is just a little too convoluted. How about

~~~ elixir
  def list_contains([], _value),             do: false
  def list_contains([value | _rest], value), do: true
  def list_contains([_ | rest], value),      do: list_contains(rest, value)
~~~